m5_forecasting/data_processor.py

```
@@ -0,0 +1,156 @@
+import pandas as pd
+import numpy as np
```

Ruff

`numpy` imported but unused

Remove unused import: `numpy`

(F401)